### DISCUSSION TITLE
How to display commands in alphabetical order....

### DISCUSSION THREAD

**Discussion by CartBlanche** [Tyrrrz/CliFx#133] [Q&A]
When I run my app with `meadow --help` 

the list of commands is not in alphabetical order. 

Is there setting or similar that will show the commands in alphabetical order when --help is used?

**Comment by Tyrrrz** [comment]
There is no setting. I think the commands are unsorted and just appear in the order they're defined:

https://github.com/Tyrrrz/CliFx/blob/a09818d4526fdb076449469b45b79b4eedf1409c/CliFx/Formatting/HelpConsoleFormatter.cs#L368-L370

You can submit a PR to change that

**Comment by CartBlanche** [comment]
@Tyrrrz I did submit a PR #134 , before seeing your reply. 
Let me know if it's ok.
